operations/configuration_management/iSCSI_SBPS_Node_Personalization.md

```diff
@@ -0,0 +1,359 @@
+# iSCSI SBPS worker node(s) personalization
+

```

Comment:
This whole file has `\r\n` line terminations, they should be just `\n`.  Please strip out all the `\r`s from this file.